src/AggregateFunctions/IAggregateFunction.h

```diff
@@ -123,6 +123,10 @@ class IAggregateFunction : public std::enable_shared_from_this<IAggregateFunctio
      */
     virtual void add(AggregateDataPtr __restrict place, const IColumn ** columns, size_t row_num, Arena * arena) const = 0;
 
+    /// Adds several default values of arguments into aggregation data on which place points to.
+    /// Default values must be a the 0-th positions in columns.
```

Comment:
Do I understand correctly that:
- default value is not dynamic, so it is always zero or an empty string, etc;
- do we have to provide the `columns` (with default values) only because there should be a convenient way to use these values?

Reply:
1. Yes, dynamic defaults are not supported now.
2. Yes, because all interfaces of aggregate functions expect columns to work with and default is always stored at 0-th positionin sparse columns, so no additional copies are required.